packages/client/src/components/Navigation/Navigation.component.js

```diff
@@ -1,53 +1,65 @@
+  const [showLinks, setShowLinks] = useState(false);
 
   return (
     <div className="navigation-container">
       <div className="navigation">
         <div className="nav">
           <span className="navbar-brand">Fashion Spree</span>
-          <NavLink to="/" className="home">
-            Home
-          </NavLink>
-          <NavLink to="/about-us" className="about-us">
-            About Us
-          </NavLink>
-          <NavLink to="/collections" className="collections">
-            Our collections
-          </NavLink>
-          <NavLink to="/contact-us" className="contact-us">
-            Contact Us
-          </NavLink>
-          {user ? (
-            <div className="logged-in-container">
-              <div className="logged-in-text">
-                <div>{name}</div>
-                <div>{user.email}</div>
+          <div className="links" id={showLinks ? 'links-mobile' : ''}>
+            <NavLink to="/" className="home">
+              Home
+            </NavLink>
+            <NavLink to="/about-us" className="about-us">
+              About Us
+            </NavLink>
+            <NavLink to="/collections" className="collections">
+              Our collections
+            </NavLink>
+            <NavLink to="/contact-us" className="contact-us">
+              Contact Us
+            </NavLink>
+            {user ? (
+              <div className="logged-in-container">
+                <div className="logged-in-text">
+                  <div>{name}</div>
+                  <div>{user.email}</div>
+                </div>
+                <div className="logout-button-container">
+                  <Button
+                    label="Logout"
+                    backgroundColor="#F5F5F5"
+                    onClick={logout}
+                  />
+                </div>
               </div>
-              <div className="logout-button-container">
-                <Button
-                  label="Logout"
-                  backgroundColor="#F5F5F5"
-                  onClick={logout}
-                />
+            ) : (
+              <div className="nav-buttons">
+                <Link to="/login" className="login">
+                  <Button label="Login" backgroundColor="#F5F5F5" />
+                </Link>
+                <Link to="/signup" className="signup">
+                  <Button label="Sign Up" backgroundColor="#F5F5F5" />
+                </Link>
               </div>
-            </div>
-          ) : (
-            <div className="nav-buttons">
-              <Link to="/login" className="login">
-                <Button label="Login" backgroundColor="#F5F5F5" />
-              </Link>
-              <Link to="/signup" className="signup">
-                <Button label="Sign Up" backgroundColor="#F5F5F5" />
-              </Link>
-            </div>
-          )}
+            )}
+          </div>
+          <button
+            onClick={() => setShowLinks(!showLinks)}
+            type="submit"
```

Comment:
`type="button"`, since it's not part of a form.

Reply:
👍🏼